deployment/k8s/texera-helmchart/files/texera_ddl.sql

```diff
@@ -264,13 +264,14 @@ CREATE TABLE IF NOT EXISTS public_project
 -- dataset
 CREATE TABLE IF NOT EXISTS dataset
 (
-    did            SERIAL PRIMARY KEY,
-    owner_uid      INT NOT NULL,
-    name           VARCHAR(128) NOT NULL,
-    is_public      BOOLEAN NOT NULL DEFAULT TRUE,
+    did             SERIAL PRIMARY KEY,
+    owner_uid       INT NOT NULL,
+    name            VARCHAR(128) NOT NULL,
+    repository_name VARCHAR(128),
+    is_public       BOOLEAN NOT NULL DEFAULT TRUE,
     is_downloadable BOOLEAN NOT NULL DEFAULT TRUE,
-    description    VARCHAR(512) NOT NULL,
-    creation_time  TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    description     VARCHAR(512) NOT NULL,
+    creation_time   TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
revert this unnecessary change

Reply:
We do NOT want to include such style changes in this PR as they have nothing to do with this PR and they can make the PR harder to review.   Please revert it as suggested by @bobbai00 .